recipes/icon/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "icon" %}
+{% set version = "9.5.2" %}
+{% set build = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gtownsend/icon/archive/refs/tags/v9.5.20i.zip
+  sha256: ed8c28b7dee31cdef95af4fffaeb88bb6c348304dba4552f41b03928907a82f3
+
+build:
+  number: 0
+  skip: True  # [win or osx or linux32]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - make
+    - sed
+    - binutils
```

Comment:
All of these should be in `build`. Why do you need `bintuils`?

Reply:
I will try again without.  Previously, my build was failing without these.

Reply:
Nope. My build fails without `make` and without `strip` from `bunutils`.  I no longer need `sed`.

Reply:
Okay. Can you move them to `build`?

Reply:
Will do. Thx.